src/main/kotlin/com/example/noswimmingserver/domain/reading_journal/service/CreateReadingJournalService.kt

```diff
@@ -0,0 +1,46 @@
+package com.example.noswimmingserver.domain.reading_journal.service
```

Comment:
🚫 **[ktlint]** <package-name> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Package name must not contain underscore (cannot be auto-corrected)